website_sale_wishlist/static/src/js/wishlist_tour.js

```diff
@@ -2,136 +2,109 @@
+                run: "click",
             },
             {
                 title: "I go to login page",
-                waitFor: ipad + " .fa-heart",
-                onload: function () {
-                    window.location = "/web/login?redirect=/shop";
-                },
+                extra_trigger: ipad + " .fa-heart",
+                trigger: "a[href='/web/login']",
+                run: "click",
             },
             {
                 title: "Fill user name and password",
-                waitFor: "#login, #password",
-                onload: function () {
-                    // HACK Weird things happen in Travis.
-                    // See https://github.com/OCA/e-commerce/pull/149
-                    $(this.waitFor).val("demo")
-                    .closest("form").submit();
-                    return "Log in";
+                trigger: "#login, #password",
+                run: function () {
+                    $('#login, #password').val("portal");
                 },
             },
             {
                 title: "Log in",
-                element: "button[type=submit]",
-                waitFor: "button[type=submit]",
-                onload: workaround_12961(
-                    "/shop",
-                    "Let me check my wishlist now that I am logged in"
-                ),
+                trigger: ".btn:contains('Log in')",
+                run: "click",
             },
             {
                 title: "Let me check my wishlist now that I am logged in",
-                element: ".js_wishlist_quantity:contains(2)",
-                waitFor: ipad + " .fa-heart",
-                waitNot: ipod + " .fa-heart",
-                wait: 1200,
-                onload: workaround_12961(
-                    "/shop?wishlist_only=1",
-                    "My wishlist is intact, I click on my name"
-                ),
+                extra_trigger: ".js_wishlist_quantity:contains(2)",
+                trigger: "a[href='/shop?wishlist_only=1']",
+                run: "click",
             },
             {
                 title: "My wishlist is intact, I click on my name",
-                element: ".dropdown-toggle:contains(Demo User)",
-                waitFor:
-                    imac + " .fa-heart, " + ipad + " .fa-heart, "
-                    + ".js_wishlist_quantity:contains(2)",
-                waitNot: ipod,
+                extra_trigger: imac + " .fa-heart, " + ipad + " .fa-heart, "
+                + ".js_wishlist_quantity:contains(2)",
+                trigger: ".dropdown-toggle:contains(Demo Portal User)",
+                run: "click",
             },
             {
                 title: "I log out",
-                element: "a[href='/web/session/logout?redirect=/']",
-                waitFor: "a[href='/web/session/logout?redirect=/']:visible",
+                extra_trigger: "a[href='/web/session/logout?redirect=/']:visible",
```

Comment:
Most likely you will not need all these `extra_trigger`s (here and in almost every step) since they are meant to display the step tooltip in a different place, but you are using the tour only for testing purposes, so you don't care about tooltips.